lib/pages/forward/forward.dart

```diff
@@ -1,50 +1,81 @@
+import 'dart:async';
+
+import 'package:dartz/dartz.dart' hide State;
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/app_state/contact/get_contacts_success.dart';
+import 'package:fluffychat/domain/usecase/fetch_contacts_interactor.dart';
+import 'package:fluffychat/mixin/comparable_presentation_contact_mixin.dart';
 import 'package:fluffychat/pages/chat_list/chat_list.dart';
+import 'package:fluffychat/pages/forward/presentation_forward.dart';
 import 'package:fluffychat/pages/forward/forward_view.dart';
 import 'package:fluffychat/utils/matrix_sdk_extensions/client_stories_extension.dart';
+import 'package:fluffychat/widgets/matrix.dart';
 import 'package:flutter/material.dart';
 import 'package:matrix/matrix.dart';
 import 'package:scroll_to_index/scroll_to_index.dart';
-import 'package:vrouter/vrouter.dart';
-
-import 'package:fluffychat/widgets/matrix.dart';
 
 class Forward extends StatefulWidget {
-  final Widget? sideView;
 
-  const Forward({Key? key, this.sideView}) : super(key: key);
+  const Forward({Key? key}) : super(key: key);
 
   @override
   ForwardController createState() => ForwardController();
 }
 
-class ForwardController extends State<Forward> {
-
-  List<Room>? rooms;
+class ForwardController extends State<Forward> with ComparablePresentationContactMixin {
 
-  Timeline? timeline;
-
-  String? get roomId => context.vRouter.pathParameters['roomid'];
+  final _fetchContactsInteractor = getIt.get<FetchContactsInteractor>();
+  final streamController = StreamController<Either<Failure, GetContactsSuccess>>();
+  final networkStreamController = StreamController<Either<Failure, GetContactsSuccess>>();
```

Comment:
what is the different? why we need 2 different stream?

Reply:
Sorry, this is duplicated,  I'll remove itt